packages/typespec-ts/test/modularUnit/modelsGenerator.spec.ts

```diff
@@ -13,6 +13,710 @@ describe("modular model type", () => {
   });
 });
 
+describe("modular encode test for property type datetime", () => {
```

Comment:
nit: one more case with utcDateTime without encode for header property.

Reply:
done